src/frontend/magnify/src/components/auth/LoginForm/LoginForm.test.tsx

```diff
@@ -37,8 +37,8 @@ const renderWithController = (controller: MockController) => {
 };
 
 const fillInForm = async (user: UserEvent, password: string) => {
-  await user.type(screen.getByRole('textbox', { name: 'Username *' }), 'username');
-  await user.type(screen.getByLabelText('Password*'), password);
+  await user.type(screen.getByRole('textbox', { name: 'Username' }), 'username');
+  await user.type(screen.getByLabelText('Password'), password);
```

Comment:
```suggestion
  await user.type(screen.getByRole('textbox', {name: 'Password'}), password);
```

Reply:
We can't because a password is not considered as a textbox